.eslintrc.js

```diff
@@ -129,6 +129,7 @@ module.exports = {
         '@typescript-eslint/no-unsafe-return': 1,
         '@typescript-eslint/restrict-plus-operands': 1,
         '@typescript-eslint/ban-ts-comment': 1,
+        'react/require-default-props': 1,
```

Comment:
Yeah, its some weird stuff going on, almost as if its trying to interpret the typescript code as normal javascript. Maybe its because of updated eslint and typescript version? 🤔